fail2ban/tests/utils.py

```diff
@@ -429,3 +436,64 @@ def assert_dict_equal(a, b):
 	assert isinstance(a, dict), "Object is not dictionary: %r" % a
 	assert isinstance(b, dict), "Object is not dictionary: %r" % b
 	assert a==b, "Dictionaries differ:\n%r !=\n%r" % (a, b)
+
+
+#
+# Server handling helpers
+#
+
+def kill_srv(pidfile):
```

Comment:
For the PR :+1: 
But, I don't like it was moved from fail2banclienttestcase.py... :-1: 
I'll move it back, together with your `@with_kill_srv` all right, at least it will be not used somewhere else (I think it will never happen).